Makefile

```diff
@@ -201,9 +201,11 @@ test-ci-js:
 		$(TEST_CI_ARGS) $(CI_JS_SUITES)
 
 test-ci: | build-addons
-	$(PYTHON) tools/test.py $(PARALLEL_ARGS) -p tap --logfile test.tap \
+	out/Release/cctest --gtest_output=tap:cctest.tap
+	$(PYTHON) tools/test.py $(PARALLEL_ARGS) -p tap --logfile jstest.tap \
 		--mode=release --flaky-tests=$(FLAKY_TESTS) \
 		$(TEST_CI_ARGS) $(CI_NATIVE_SUITES) $(CI_JS_SUITES)
+	$(PYTHON) tools/merge-tap.py test.tap cctest.tap jstest.tap
```

Comment:
I thought that might be the case.  I'll remove the script again, it was really just to sanity-check the gtest changes.